pages/index.tsx

```diff
@@ -42,7 +43,98 @@ const HomePage: ThirdwebNextPage = () => {
           <HeroSection />
           <PartnerCarousel />
           <StatsSection />
-          <HowItWorksSection />
+          <HomePageCard
+            title="Onboard everyone to web3, instantly"
+            description="The complete web3 wallet stack — with a customizable Connect Wallet modal, embedded wallets for flexible sign-in options, and account abstraction for the smoothest user experiences."
+            introductionTitle="WALLET PRODUCTS"
+            image={require("public/assets/bear-market-airdrop/desktop-wallets.png")}
+            mobileImage={require("public/assets/bear-market-airdrop/mobile-wallets.png")}
+            items={[
+              {
+                title: "Connect",
+                description: "Fully customizable Connect Wallet component",
```

Comment:
please let's not hardcode these we have a centralized place for these in `components/product-pages/common/nav/data.ts`, otherwise we need to maintain many things at once